MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,47 @@
+# Migration guide
+
+## From 0.4.0 to 0.5.0
+
+**Step 1**: First of all you need to make sure you upgraded Steiger package to version 0.5.0 or higher.
+
+**Step 2**: You need to install `@feature-sliced/steiger-plugin` package (that contains all FSD rules to run checks in your project). Run one of the following commands based on the package manager you use.
+
+**pnpm**:
+
+```shell
+pnpm add @feature-sliced/steiger-plugin
+```
+
+**yarn**:
+
+```shell
+yarn add @feature-sliced/steiger-plugin
+```
+
+**npm**:
+
+```shell
+npm i @feature-sliced/steiger-plugin
+```
+
+**Step 3**: The final step. You need to bring your `steiger.config.js` file to the new configuration format. You can do that automatically by running one of the following commands.
```

Comment:
suggestion: since the migration script is imperfect (and the command is also long), I'd like to include a bit about what it actually does. For example, show a "before" and "after". And also, if people have a `steiger.config.ts` instead of `.js`, we should let them know that they need to modify this command.

As for the "before" and "after", we can do it like this in Markdown:

    <table><thead><tr>
    <th>Before</th>
    <th>After</th>
    </tr></thead><tbody><tr><td>
    
    ```ts
    // steiger.config.ts
    ```
    
    </td><td>
    
    ```ts
    // steiger.config.ts
    ```
    
    </td></tr></tbody></table>

The empty lines are there to prevent HTML from messing up Markdown. It will produce this:

<table><thead><tr>
<th>Before</th>
<th>After</th>
</tr></thead><tbody><tr><td>

```ts
// steiger.config.ts
```

</td><td>

```ts
// steiger.config.ts
```

</td></tr></tbody></table>